docs/simple_switch.md

```diff
@@ -685,3 +685,12 @@ you have two actions for table `t` where one has the action
 `m.read(result_field2)`, or if both of those calls are in the same
 action.  All calls to the `read()` method for `m` must have the same
 result parameter where the result is written.
+
+### BMv2 `meter rate units` implementation notes
```

Comment:
what is this documentation for? The runtime CLI? The Thrift service? The PD API?

Reply:
simple_switch_CLI

Reply:
that's not accurate then
IIRC, the burst size is in "units" (packets or bytes) and the rate is in "units per microsecond" (https://github.com/p4lang/behavioral-model/blob/master/thrift_src/standard.thrift#L88)

Reply:
Wow, even I quoted the same line of code in the Slack channel today.  But, seeing the code in pd_helpers.cpp, I saw the conversion from kbps and kbits.  This cpp code also happens to be the only code in the behavioral-model repo converting rate and burst_size and thus I assumed the code is used with simpe_switch_CLI as well.

Reply:
> that's not accurate then
> IIRC, the burst size is in "units" (packets or bytes) and the rate is in "units per microsecond" (https://github.com/p4lang/behavioral-model/blob/master/thrift_src/standard.thrift#L88)

There is some confusion.  Based on your reply for meter type as packet, its "units in microsecond" and this is the relevant code in pd_helpers.cpp. I see the division by 10^6 for microseconds.

`info_rate = static_cast<double>(meter_spec->cir_pps) / 1000000.;`

Isn't the user input captured by simple_switch_CLI in meter_spec->cir_pps and thus the user inputs in pps units?  Or is the code in pd_helpers.cpp not used at all by simple_switch_CLI?

Reply:
The PD layer is not used at all by the CLI. If you look at the CLI code, you will see that it calls the Thrift service directly.
It is fine to have this information in simple_switch.md BTW, but it would be far more useful to improve the runtime_CLI help message for the `meter_set_rates` CLI command, so that users can get this information directly from the CLI a they are using it.

Reply:
Ok, I will edit the simple_switch.md doc for CLI rates and burst_size.  Even, I was thinking of adding the help message for the meter_set_rates.  I will add that.  Thanks!